src/clientRoutes.js

```diff
@@ -407,7 +407,14 @@ const handleV2CoinSpecificREST = function(req, res, next) {
     return redirectRequest(bitgo, method, coinURL, req, next);
   } catch (e) {
     if (e instanceof errors.UnsupportedCoinError) {
```

Comment:
So this is meant to catch the case when params.coin is left empty or its an unsupported coin?

Reply:
Yes, it handles the case where we got a request for a route which expects a coin, tries to instantiate a coin from params.coin, and fails to do so. In that case, we fall back trying the route as a coinless route instead of immediately failing.